setup.py

```diff
@@ -19,26 +30,6 @@ def get_test_requires():
   else:
     return []
 
-
-class Test(TestCommand):
```

Comment:
This line - https://github.com/opendatahub-io/jupyterhub-singleuser-profiles/blob/993bc59479c0253f78c8c6e00073dc1090bd0372/setup.py#L52 - has to be deleted as well